client/my-sites/stats/stats-chart-tabs/index.jsx

```diff
@@ -219,6 +221,12 @@ module.exports = React.createClass( {
 			activeTabLoading = this.props.activeTabVisitsList.isLoading() && this.props.visitsList.isLoading(),
 			classes;
 
+		analytics.tracks.recordEvent( 'calypso_stats_chart_tab_view', {
```

Comment:
Why not use [TrackComponentView](https://github.com/Automattic/wp-calypso/tree/master/client/lib/analytics/track-component-view)? I'd probably move the nudge code into a component method, like `this.buildNudge()` and then have it return null for premium/business plans sites, the nudge for the "on" test, and `<TrackComponentView/>` for the "off" test.